misk-clustering-dynamodb/src/main/kotlin/misk/clustering/dynamo/DyClusterMember.kt

```diff
@@ -0,0 +1,15 @@
+package misk.clustering.dynamo
+
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbAttribute
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey
+
+@DynamoDbBean
+class DyClusterMember {
+  @get:DynamoDbPartitionKey
+  var name: String? = null
+
+  // Stored as epoch ms
+  @get:DynamoDbAttribute("updated_at")
+  var updated_at: Long? = null
```

Comment:
I might prefer to keep `pod_name` as an additional non-functional metadata, similar to https://github.com/squareup/cash-dynamodb-lease/blob/48fd2dbe64e28f9c3a5fd7ab280b00ce2e83b4de/lib/src/main/kotlin/com/squareup/cash/dynamodb/lease/RealDynamoDbLease.kt#L153
and keep `name` as UUID. This would allow us to run multiple cluster clients per pod as well if we wanted to do that for some reason (not saying we should)